src/Tizen.NUI/src/public/Widget/WidgetView.cs

```diff
@@ -685,6 +704,44 @@ private PropertyMap InternalEffect
             }
         }
 
+        /// <summary>
+        /// Gets or sets KeepWidgetSize
+        ///
+        /// if this value is true, WidgetView keep widget instance's size and don't resize even if WidgetView is resized.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
```

Comment:
I think we can remove this since_tizen tag.
And as you know, the current latest version is 12 :crying_cat_face:

Reply:
Thank you. i will remove it!